app/views/job_invocations/_tab_overview.html.erb

```diff
@@ -1,59 +1,29 @@
-  <%= host_counter(_('Total hosts'), job_invocation.total_hosts_count) %>
-</div>
-
-<div class="col-md-6 infoblock">
-  <h4><%= _('Providers and templates') %></h4>
-  <% job_invocation.pattern_template_invocations.each do |template_invocation| %>
-    <h5>
-      <b><%= template_invocation.template.name %></b> <%= 'through' %> <%= template_invocation.template.provider.humanized_name %>
-    </h5>
-    <% target = template_invocation.targeting.hosts.with_os.first || template_invocation.targeting.hosts.first %>
-    <%= _('Preview for target %s') % target.try(:name) || 'N/A' %>
-
-    <%= preview_box(template_invocation, target) %>
-
+<% template_invocations = job_invocation.pattern_template_invocations %>
+<div class='row'>
+  <div class="col-xs-6 col-sm-6 col-md-6">
+    <%= render :partial => 'card_results' %>
+    <%= render :partial => 'card_schedule', :locals => { :job_invocation => job_invocation } %>
+  </div>
+  <div class='col-xs-12 col-sm-6 col-md-6'>
+  <%= render :partial => 'card_target_hosts', :locals => { :job_invocation => job_invocation } %>
+  <% template_invocations.each do |template_invocation| %>
+    <%= minicard('user', template_invocation.effective_user || Setting[:remote_execution_effective_user],
+                 template_invocation.template.name + ' ' + _('effective user')) %>
+    <%= minicard('cluster', job_invocation.total_hosts_count, _('Total hosts')) %>
     <% if template_invocation.input_values.present? %>
-      <%= _('following user inputs were provided') %>
-      <ul>
-        <% template_invocation.input_values.joins(:template_input).each do |input_value| %>
-          <li><b><%= input_value.template_input.name %></b>: <%= trunc_with_tooltip(input_value.value, 255) %></li>
-        <% end %>
-      </ul>
-    <% end %>
-    <% if template_invocation.effective_user %>
-      <b><%= _("Effective user") %></b>: <%= template_invocation.effective_user %><br>
-    <% end %>
-    <% if job_invocation.concurrency_level %>
-      <b><%= _("Concurrency level limited to") %></b>: <%= job_invocation.concurrency_level %> <%= _('tasks at a time') %><br>
-    <% end %>
-    <% if job_invocation.time_span %>
-      <b><%= _("Set to distribute over") %></b>: <%= job_invocation.time_span %> <%= _('seconds') %><br>
-    <% end %>
-    <% if job_invocation.task && (job_invocation.task.delayed? || job_invocation.task.recurring?) %>
-      <b><%= _("Scheduled to start at") %></b>: <%= job_invocation.task.start_at.try(:in_time_zone) %><br>
-    <% end %>
-    <% unless job_invocation.execution_timeout_interval.nil? %>
-      <b><%= _("Timeout to kill after") %></b>: <%= job_invocation.execution_timeout_interval %> <%= _('seconds') %><br>
+      <%= render :partial => 'card_user_input', :locals => { :template_invocation => template_invocation } %>
     <% end %>
   <% end %>
+  </div>
 </div>
+<hr/>
+<% if job_invocation.resolved? %>
+  <div id="hosts" data-refresh_required="<%= job_invocation.resolved? ? '' : 'true' %>">
```

Comment:
@iNecas @adamruzicka The table went back to the 1st page after every reload because on a previous update, this was set to 'true' all the time. I've changed it to this conditional and now it's fine.